common/tools/dev-tool/src/commands/samples/tsToJs.ts

```diff
@@ -49,7 +49,10 @@ const REGEX_STACK: Array<[RegExp, string]> = [
   [/import\s+({[^}]+})\s+from\s*("[^"]+");/gs, "const $1 = require($2);"],
```

Comment:
I did this for the imports. The exports are easier to process like this, since non-modifier based exports (`export =`, `export {}`, and `export from`) are not supported currently. We might try to support these in the future if there is a need, but samples that reach this level of complexity probably belong off-repo somewhere in Azure-Samples.